TIL/topics/dart_basics/AsyncAndSync.md

```
@@ -0,0 +1,86 @@
+		}
+		void main() {
+			readData().then((data) => print(data));
+		}
+		```
+		- 실행 순서나 완료의 예측이 어려운 단점이 있다.
+	- **Future / async와 await 사용**
+		- 비동기 코드를 순차적으로 실행하게 할 수 있다.
+		- 디버깅이 쉽고 예측이 가능하다.
+		- 비동기로 작업을 진행하기 때문에 성능은 떨어질 수 있다.
+		```dart
+		Future<String> readData() async {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)